ansible/roles/openshift_setup/tasks/main.yml

```diff
@@ -200,11 +200,11 @@
 
   - name: Check to see if we need to use a custom config
     set_fact:
-      use_custom_config: "{{use_ssl}}"
+      use_custom_config: '{{ use_ssl }}'
 
   - name: Check to see if we need to regenerate the custom config because something is missing
     set_fact:
-      generate_config: "{{use_custom_config and (not master_config_stat.stat.exists or not console_ssl_stat.stat.exists)}}"
+      generate_config: '{{(use_custom_config or openshift_build_type == "mac") and (not master_config_stat.stat.exists or not console_ssl_stat.stat.exists)}}'
```

Comment:
This also needs to run ```when: broker_ci```. Agree with Chris that another variable to cover this would be ideal.

Reply:
Clarification.  When running CI,  oc cluster up needs to run to generate the config.
```(use_custom_config or openshift_build_type == "mac" or broker_ci)```